mailer-app/routes/profile.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const router = express.Router();
+const User = require("../models/User");
+
+
+//Iteration 4 - Profile Routes
+router.get("/:confirmCode", (req, res) => {
+    const { confirmCode } = req.params;
+    User.findOne({ confirmationCode: confirmCode })
```

Comment:
aqui ibas a hacer un find and update para cambiar el valor predeterminado del status a activo para validar al usuario